pages/resources/collectibles.mdx

```diff
@@ -0,0 +1,802 @@
+
+###### Example Profile Effect Config
+
+```json
+{
+  "type": 1,
+  "id": "1139323075519852625",
+  "sku_id": "1139323092645183591",
+  "title": "Hydro Blast",
+  "description": "Time to make a splash.",
+  "accessibilityLabel": "A powerful stream of water swirls and twirls in the air. Wait, where's it headed off to?!",
+  "animationType": 2,
+  "thumbnailPreviewSrc": "https://cdn.discordapp.com/assets/profile_effects/effects/b17d139f2e9/splash/thumbnail.png",
+  "reducedMotionSrc": "https://cdn.discordapp.com/assets/profile_effects/effects/b17d139f2e9/splash/reducedMotion.png",
+  "effects": [
+    {
+      "src": "https://cdn.discordapp.com/assets/profile_effects/effects/b17d139f2e9/splash/intro.png",
+      "loop": false,
+      "height": 880,
+      "width": 450,
+      "duration": 2880,
+      "start": 0,
+      "loopDelay": 0,
+      "position": {
+        "x": 0,
+        "y": 0
+      },
+      "zIndex": 100,
+      "randomizedSources": []
+    },
+    {
+      "src": "https://cdn.discordapp.com/assets/profile_effects/effects/b17d139f2e9/splash/loop.png",
+      "loop": true,
+      "height": 880,
+      "width": 450,
+      "duration": 2880,
+      "start": 5760,
+      "loopDelay": 5760,
+      "position": {
+        "x": 0,
+        "y": 0
+      },
+      "zIndex": 100,
+      "randomizedSources": []
+    }
+  ]
+}
+```
+
+<RouteHeader method="GET" url="/users/@me/valid-collectibles-gift-recipient">
+  Get Valid Collectibles Gift Recipient
+</RouteHeader>
+
+Returns [gift eligibility](#gift-eligibility-structure) object for a given user for given collectible SKU ID.
+
+###### Query String Params
+
+| Field        | Type      | Description                                      |
+| ------------ | --------- | ------------------------------------------------ |
+| recipient_id | snowflake | The ID of the user to check for gift eligibility |
+| sku_id       | snowflake | The SKU ID of the collectible to gift            |
+
+<RouteHeader method="GET" url="/users/@me/valid-collectibles-gift-recipients-batch">
+  Get Valid Collectibles Gift Recipients Batch
+</RouteHeader>
+
+Returns a mapping of SKU IDs to [gift eligibility](#gift-eligibility-structure) object for a given user for given collectible SKU IDs.
+
+###### Query String Params
+
+| Field        | Type             | Description                                      |
+| ------------ | ---------------- | ------------------------------------------------ |
+| recipient_id | snowflake        | The ID of the user to check for gift eligibility |
+| skus_id      | array[snowflake] | The SKU IDs of the collectibles to gift          |
+
+<RouteHeader method="PUT" url="/users/@me/claim-premium-collectibles-product">
+  Claim Premium Collectibles Product
+</RouteHeader>
+
+Returns list of [collectible products](#collectible-product-structure) objects owned by the current user if the user hasn't claimed the product yet, otherwise returns 204 empty response.
```

Comment:
```suggestion
Claims a collectible SKU provided for free to premium users. Returns a list of [collectible products](#collectible-product-structure) objects owned by the current user if the user hasn't claimed the product yet, otherwise returns a 204 empty response.
```
you didn't specify what the endpoint actually does